### ISSUE TITLE
filterDoublets error in as.POSIXct.numeric(e)

### ISSUE THREAD

**Issue by JamesDanielOReilly** [GreenleafLab/ArchR#396]
**Describe the bug**
Calling `filterDoublets()` gives error "Error in as.POSIXct.numeric(e) : 'origin' must be supplied". This bug was previously addressed in issue #217 . @jgranja24  said that this is now fixed on the release branch but I'm still running into this problem even when pulling from this branch. The issue arises both on the master branch and the 1.0.0 release branch. I have attached screenshots of the console output.

**To Reproduce**
Follow the steps given in the tutorial. The issue arises when `filterDoublets()` is called. The issue also arises with my own dataset, where I'm following the exact steps given in the tutorial.

**Session Info**
I'm running R 4.0.3 on ubuntu 18.04, using RStudio. I'm not certain if ArchR has yet been tested with R >v4.0.0, so perhaps this is where the issue lies. I've included my session info in a screenshot also.
![issue](https://user-images.githubusercontent.com/44339381/98118424-7cfb0a80-1eab-11eb-9e60-c28987788a34.png)
![sessioninfo](https://user-images.githubusercontent.com/44339381/98118426-7d93a100-1eab-11eb-8221-4fc13babeeb5.png)

**Comment by rcorces** [comment]
from what I can tell, it looks like no change was implemented to fix #217 
@jgranja - I still see the problematic `addProjectSummary()` call in the current master version. I thought the fix was supposed to remove this because of issues with `Sys.time()` in R v4.*?

https://github.com/GreenleafLab/ArchR/blob/b6b42107098ecdfafc064855bcefe32844a0027c/R/FilterCells.R#L101

**Comment by JamesDanielOReilly** [comment]
> from what I can tell, it looks like no change was implemented to fix #217
> @jgranja - I still see the problematic `addProjectSummary()` call in the current master version. I thought the fix was supposed to remove this because of issues with `Sys.time()` in R v4.*?
> 
> https://github.com/GreenleafLab/ArchR/blob/b6b42107098ecdfafc064855bcefe32844a0027c/R/FilterCells.R#L101

For now, should I implement the modified function you referenced in #217 ?

**Comment by rcorces** [comment]
yes - that should solve your problem

**Comment by sylestiel** [comment]
I'm having the same issue as well (Error in as.POSIXct.numeric(e) : 'origin' must be supplied) despite installing
devtools::install_github("GreenleafLab/ArchR", ref="release_1.0.0", repos = BiocManager::repositories()) 

So in essence i don't believe the doublets are being filtered

**Comment by lzj1769** [comment]
Same issue as well.
I am running ArchR on our HPC cluster.

**Comment by sylestiel** [comment]
Issue addressed by reverting back to R version 3.6.3 (2020-02-29).

**Comment by jgranja24** [comment]
This but should be fixed now in the new release branch. I removed the timestamps recording and I believe this works on R 4.0.

`devtools::install_github("GreenleafLab/ArchR", ref="release_1.0.1", repos = BiocManager::repositories())
`

On R -
R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.5

```
filterDoublets(proj)
Filtering 49 cells from ArchRProject!
	PBMCSmall : 49 of 2217 (2.2%)

           ___      .______        ______  __    __  .______      
          /   \     |   _  \      /      ||  |  |  | |   _  \     
         /  ^  \    |  |_)  |    |  ,----'|  |__|  | |  |_)  |    
        /  /_\  \   |      /     |  |     |   __   | |      /     
       /  _____  \  |  |\  \\___ |  `----.|  |  |  | |  |\  \\___.
      /__/     \__\ | _| `._____| \______||__|  |__| | _| `._____|
    
class: ArchRProject 
outputDirectory: /Users/jeffreygranja/Desktop/PBMCSmall 
samples(1): PBMCSmall
sampleColData names(1): ArrowFiles
cellColData names(18): Sample TSSEnrichment ... ReadsInPeaks FRIP
numberOfCells(1): 2168
medianTSS(1): 12.178
medianFrags(1): 588
```
